src/redux/missions/missions.js

```diff
@@ -1,12 +1,34 @@
-// const URL = 'https://api.spacexdata.com/v3/missions';
+const URL = 'https://api.spacexdata.com/v3/missions';
+const GET_MISSIONS = 'spaceTraveler/missions/GET_MISSIONS';
 
 const initialState = [];
 export default function reducer(state = initialState, action) {
-  const { type } = action;
+  const { type, payload } = action;
   switch (type) {
-    // case GET_MISSIONS:
-    //   return payload.missions;
+    case GET_MISSIONS:
+      return payload.missions;
```

Comment:
Here use `[...state, ...payload.missions]` to ensure you maintain the initial state

Reply:
Ok